packages/terra-tag/src/Tag.scss

```diff
@@ -0,0 +1,73 @@
+
+:local {
+  .tag {
+    align-items: center;
+    background-color: var(--terra-tag-background-color, rgba(28, 31, 33, 0.04));
+    border-color: var(--terra-tag-border-color, rgba(28, 31, 33, 0.1));
+    border-radius: 0.2142rem;
+    border-style: var(--terra-tag-border-style, solid);
+    border-width: 0.07142rem;
+    box-shadow: var(--terra-tag-box-shadow, none);
+    color: var(--terra-tag-color, #434a4d);
+    display: inline-flex;
+    font-size: var(--terra-tag-font-size, 1rem);
+    font-weight: var(--terra-tag-font-weight, normal);
+    line-height: var(--terra-tag-line-height, 1.14285);
+    margin-bottom: var(--terra-tag-margin-bottom, 0.357rem);
+    margin-left: var(--terra-tag-margin-left);
+    margin-right: var(--terra-tag-margin-right, 0.714rem);
+    margin-top: var(--terra-tag-margin-top, 0.357rem);
+    padding-bottom: var(--terra-tag-padding-bottom, 0.1428rem);
+    padding-left: var(--terra-tag-padding-left, 0.5rem);
+    padding-right: var(--terra-tag-padding-right, 0.5rem);
+    padding-top: var(--terra-tag-padding-top, 0.1428rem);
+    text-align: left;
+    text-decoration: var(--terra-tag-text-decoration, none);
```

Comment:
This could be simplified to 
```
text-decoration: var(--terra-tag-text-decoration);
```

Reply:
updated it [here](https://github.com/cerner/terra-core/pull/1306/commits/6506e16da9f3b3d557d3c71bb7fa03a4874a99a9).